### PR TITLE
Replaced cluster name using integration id in alert messages

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@nthomas-redhat cluster name is replaced by integration id

**Comment by GowthamShanmugam** [comment]
@gnehapk can you please raise an issue with detailed  info of problem which you are facing in UI

**Comment by GowthamShanmugam** [comment]
@r0h4n @shtripat please review

**Comment by rishubhjain** [comment]
@GowthamShanmugam Attach bug id

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py [hunk 1]
```diff
@@ -38,13 +38,13 @@ def format_alert(self, alert_json):
...
+                    alert['tags']['integration_id'],
                     alert['current_value'],
                     alert['severity'],
                     alert['tags']['warning_max']
                 ))
             elif alert['severity'] == "INFO":
-                alert['tags']['message'] = ("Cluster utilization of cluster %s is"\
+                alert['tags']['integration_id'] = ("Cluster utilization of cluster %s is"\
                 " back to normal" % (
                     alert['tags']['cluster_name']
```
Comment (rishubhjain) [inline tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py]: @GowthamShanmugam I feel there is a requirement to replace cluster_name over here as well

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py [hunk 2]
```diff
@@ -38,13 +38,13 @@ def format_alert(self, alert_json):
...
                 alert['tags']['message'] = ("Cluster utilization of cluster %s is" \
                 " %s which is above the %s threshold (%s)." % (
-                    alert['tags']['cluster_name'],
+                    alert['tags']['integration_id'],
                     alert['current_value'],
                     alert['severity'],
                     alert['tags']['warning_max']
                 ))
             elif alert['severity'] == "INFO":
-                alert['tags']['message'] = ("Cluster utilization of cluster %s is"\
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py]: @rishubhjain in ui they are filter alerts using cluster id so if alert is displayed with integration id then they can under stand easily.

[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py]: ah my mistake, i will change it

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py (hunk 1 above)
Comment (GowthamShanmugam) [reply tendrl/monitoring_integration/alert/handlers/cluster/gluster/cluster_utilization_handler.py]: ya i have wrongly renamed

**Comment by GowthamShanmugam** [comment]
@rishubhjain changes done

**Comment by shtripat** [comment]
Are these changes verified with UI filtering ?

**Comment by GowthamShanmugam** [comment]
@gnehapk i am sending integration id in tags as you told but in ui it is filtering only using cluster_id, so are you going to change it?

**Comment by gnehapk** [comment]
@GowthamShanmugam In all the API responses,  cluster_id and integration_id are same for gluster clusters and same confirmed by @anivargi . Hence, UI will use cluster_id to do filtering.

**Comment by rishubhjain** [comment]
LGTM @r0h4n Please review

**Comment by GowthamShanmugam** [comment]
@r0h4n please review

**Pull request merged**